### ISSUE TITLE
:bug: Year updation to be done 

### ISSUE THREAD

**Issue by khareyash05** [konveyor/tackle2-ui#1007]
On seeing the info section of Konveyor , Copyright @ 2022 is mentioned. First of all it needs to be updated and I would like to replace the regular update by adding `Date.getFullYear()`. this will update the year automatically. Following is the screenshot
![image](https://github.com/konveyor/tackle2-ui/assets/60147732/9fe16938-7bd7-4b61-a625-bf0fb2fef2b6)

**Comment by khareyash05** [comment]
> Looking good @khareyash05 - There is one small issue RE: tests passing on your PR. Otherwise LGTM! Thanks for helping with out with this.

Thank you but this was just a small issue. I am not able to understand from the CI what is the error as all I have changed is the format of the Date. Could you pls help me with that?

**Comment by ibolton336** [comment]
@khareyash05 To run the tests locally, you can run 

```npm run test -w client -- --watch```

And then, press the `u` key to update any failed tests locally.

**Comment by ibolton336** [comment]
<img width="769" alt="Screenshot 2023-07-05 at 1 48 27 PM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/d562897f-4c7e-4ea1-95a7-c734a51241e9">

**Comment by khareyash05** [comment]
> @khareyash05 To run the tests locally, you can run
> 
> `npm run test -w client -- --watch`
> 
> And then, press the `u` key to update any failed tests locally.

Thank you will do that

**Comment by ibolton336** [comment]
Bonus points if you can hard set a timezone for the snapshot tests to run in as a part of this PR ( as @mturley suggested) 
Something similar to https://medium.com/front-end-weekly/jest-snapshot-testing-with-dates-and-times-f3badb8f1d87

**Comment by mturley** [comment]
I think a time zone wouldn't help because the issue is what year the test was run in when the snapshot was created. You'd have to simulate a specific clock time when running the snapshots. Probably doable?

@ibolton336 in the example you linked the test passes in a specific date/time as a prop when rendering the component for the snapshot, which we could totally do (and in production it would just always default to the current date)

**Comment by khareyash05** [comment]
> I think a time zone wouldn't help because the issue is what year the test was run in when the snapshot was created. You'd have to simulate a specific clock time when running the snapshots. Probably doable?
> 
> @ibolton336 in the example you linked the test passes in a specific date/time as a prop when rendering the component for the snapshot, which we could totally do (and in production it would just always default to the current date)

Can I take sometime for this? @mturley @ibolton336 I am actually occupied with the Hub tests right now and will do that probably in the weekend.

**Comment by mturley** [comment]
Of course! We could also just update the existing text to say 2023 and come back to it. Either way it's not a priority, no worries

**Comment by dymurray** [comment]
Closing due to inactivity or age, please reopen if you feel this was done in error
